### DISCUSSION TITLE
Tiling for seamless textures!

### DISCUSSION THREAD

**Discussion by PatronNick** [lllyasviel/stable-diffusion-webui-forge#277] [Q&A] [answered]
Help make "Tiling" work (for seamless textures) on Forge!
Works on 1111, can't make on Forge (((

**Comment by mingzhon** [comment]
Yes, I thought it was a setup issue, hopefully it's fixed

**Comment by catboxanon** [comment, marked as answer]
This was re-implemented in https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/5166a723c28c35b85e3f086eb79cf061f775a1b8

**Comment by jefferysanders** [comment]
I originally reported this, and confirm its fixed.

**Reply by chefexperte** [reply]
Can you explain how to use it? In Automatic I just had to check the box, now with forge I could find "tiling" in the settings, but enabling it doesn't do anything.

**Reply by jefferysanders** [reply]
The method you are using is correct. It's broken again since the release of Flux compatibility.  You either have to use a pre-flux forge or AUI for now.
